src/SystemAbstractions/SystemAbstractions.csproj

```diff
@@ -9,7 +9,15 @@
   <Import Project="..\..\build\NetStandardRelease.targets" />
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.9.5-beta1.final">
```

Comment:
I'm wondering if these additions should be in build/NetStandardTest.targets.

Reply:
Per our conversation, removed from NetStandardRelease.targets and leaving them where VS tooling can update them